### DISCUSSION TITLE
[Question]: Markdown style formating

### DISCUSSION THREAD

**Discussion by ScottMGerstl** [ngneat/transloco#458] [Q&A]
Is there any way, currently, I can accomplish setting formatters to parse a markdown-style syntax to create rich html.

The goal is to be able to create content with proper semantic markup for screen readers and SEO leveraging inline text markup such as `<strong>` and `<abbr>`

For instance: 

with a key/value of
```json
{
    "my.rich.string": "This is ##thee## richest text you've __ever__ seen. [And I've seen a lot](https://www.somelink.com)"
}
```

I would like 
```html
<p>{{ t('my.rich.string') }}</p>
```

to output
```html
<p>This is <strong>thee</strong> richest text you've <span class="app-underline">ever</span> seen. <a href="https://www.somelink.com">And I've seen a lot</a></p>
```
I would obviously have to assign it to innerHTML but you get the gist of it, I hope.

I would ideally like to do the transformation on load of the i18n files (post consolidation) so I don't have to interpret the markdown multiple times

If it doesn't exist, I'd be happy to look into issuing a PR if you can point me in the right direction

Thanks!

**Comment by ScottMGerstl** [comment]
Looks like I may have found a solution in [this plugin](https://www.npmjs.com/package/ngx-transloco-markup)

**Reply by itayod** [reply]
Great, you can also use the `innerHTML` of an element like we wrote in the [docs](https://ngneat.github.io/transloco/docs/faq#can-i-use-html-markings-inside-a-translation)
